index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <title>
+    Ariel's Portfolio
+    </title>
+    <section id="one">
+        <img class= "logo alignment" src="avatar.jpg">
```

Comment:
not sure if the logo class is necessary could just use 
#homeContent img {
//styles
}